pyflow/RCDT2/Nodes/core.py

```diff
@@ -0,0 +1,183 @@
+# SPDX-FileCopyrightText: Alliander N. V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from PyFlow.Core import NodeBase, PinBase
+import rclpy
+from rclpy.node import Node
+from threading import Thread
+from time import sleep
+from logging import getLogger
+from inflection import underscore
+
+logger = getLogger(__name__)
+
+
+class PinManager:
+    def __init__(self):
+        self.input_pins: dict[str, PinBase] = {}
+        self.output_pins: dict[str, PinBase] = {}
+
+    def add_input_pin(self, pin_name: str, pin: PinBase) -> None:
+        self.input_pins[pin_name] = pin
+
+    def add_output_pin(self, pin_name: str, pin: PinBase) -> None:
+        self.output_pins[pin_name] = pin
+
+    def get_data(self, pin_name: str) -> object:
```

Comment:
Hier gebruiken we `object`  als type hint. `typing.Any ` is voor type hints beter als je het type niet weet, mypy houdt vaak niet zo van object.